src/cart/crt_iv.c

```diff
@@ -1418,6 +1419,7 @@ crt_hdlr_iv_fetch_aux(void *arg)
 			D_GOTO(send_error, rc);
 		}
 	} else {
+		output->ifo_misc = iv_value;
```

Comment:
i dont think we should be overriding iv_value like this. instead i was thinking ivo_on_fetch() should have new argument that it can return 'misc info' which it can populate on failure if it wants and which would be assigned here.
If we fail for some reason in ivo_on_get/put parts and get here, iv_value will be not what you expect.

Reply:
Actually the user only check iv_value for certain failure. as long as on_get/put does not return the same failure. it might be ok.

But this is not good indeed. hmm.